wheel/src/lib.rs

```diff
@@ -53,6 +90,38 @@ pub async fn bootstrap_wheel(
         .map_err(Error::err)
 }
 
+pub fn test_prometheus() {
+    tokio::spawn(async move {
+        log::info!("every second update example_num and exampe_time");
+        loop {
+            EXAMPLE_COUNTER.inc();
+            let dt = Local::now();
+            let t = dt.timestamp_millis();
+            EXAMPLE_TIME_GAUGE.set(t as f64);
+            sleep(tokio::time::Duration::from_millis(10)).await;
+        }
+    });
+}
```

Comment:
Why we need test here?

1. Tests should not be placed in `lib.rs`, it's only used for building and bootstrapping nodes.
2. I think we don't need to unit test to test if prometheus works. Besides, this test cannot give you any useful feedbacks.
3. If you want to add a unit test, Please use `#[cfg(test)]` `#[test]` and wrap them in `tests` module to make it only available when testing. Please refer to the Rust book.